internal/data/balances.go

```diff
@@ -23,7 +20,7 @@ type BalancesQ interface {
 	New() BalancesQ
 	Insert(Balance) error
 	UpdateAmountBy(points int64) error
-	SetPassport(hash string, exp time.Time, isWithdrawalAllowed bool) error
+	SetIsWithdrawalAllowed(isWithdrawalAllowed bool) error
```

Comment:
we don't need this method, because proof is verified each time; also let's now drop outdated fields, like passport_hash, expires and withdrawal